test/modules/mongodb_cluster.go

```diff
@@ -0,0 +1,126 @@
+package modules
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+
+	aws_sdk "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas-sdk/v20231001002/admin"
+)
+
+func DeployMongoDBCluster(t *testing.T, workingDir string) {
+	// Get the role arn from env
+	roleArn := os.Getenv("TF_VAR_mongodb_role_arn")
```

Comment:
This isn't needed. Whenever a variable is preprended by TF_VAR_ it is automatically injected into terraform. So locally if we use the run_tests.py it sets up the environment variable, and in CI we define the ENV in the command. So this should work without getting the ENV using os.Getenv and we don't need to pass it in the Vars map.